docs/maui/behaviors/select-all-text-behavior.md

```diff
@@ -0,0 +1,83 @@
+---
+title: SelectAllTextBehavior - .NET MAUI Community Toolkit
+author: vhugogarcia
+description: "The SelectAllTextBehavior is an effect that will select all text in an Entry / Editor when it becomes focused."
```

Comment:
Is it an effect or a behavior? I see the terms are being mixed in this doc.

Also I think behaviors are not a thing anymore in .NET MAUI?